_studio/mfx_lib/shared/include/mfx_h264_enc_common_hw.h

```diff
@@ -942,6 +942,15 @@ namespace MfxHwH264Encode
 
     ENCODE_FRAME_SIZE_TOLERANCE ConvertLowDelayBRCMfx2Ddi(mfxU16 type, bool bTCBRC);
 
+    enum class SliceDividerType
+    {
+        OneSlice = 0,
```

Comment:
Imho, that's a weird to see such a mix of platforms, use cases and slice types! It blows my mind when we are supposed to use what? It depends on platform, on usage or on what? @mgonchar : probably not in this commit, but I wonder, can this somehow be improved?